internal/cmdsearchreplace/cmdsearchreplace.go

```diff
@@ -0,0 +1,91 @@
+package cmdsearchreplace
```

Comment:
I think we should move this to the `commands` package.

Reply:
This is the pattern we are following for all other commands. I don't think we should have it as an anomaly. The other option is to refactor all the other commands in a single different PR.